crates/relayer/Cargo.toml

```diff
@@ -23,7 +23,11 @@ telemetry = ["ibc-telemetry"]
 ibc-proto         = { version = "0.39.0", features = ["serde"] }
 ibc-telemetry     = { version = "0.26.4", path = "../telemetry", optional = true }
 ibc-relayer-types = { version = "0.26.4", path = "../relayer-types", features = ["mocks"] }
+osmosis-std = "0.21.0"
```

Comment:
For the sake of our already huge dependency graph, instead of depending on `osmosis-std` for a single proto (`DecProto`), can we copy-paste the proto definition in the same file where it's used?

And let's do the same with `cosmwasm_std::{Decimal, Uint128}`